articles/network-types-topologies/index.md

```diff
@@ -0,0 +1,91 @@
+
+They are categorized based on their:
+
+-  Geographical spread of network strength.
+-  Type of usage and applications.
+-  Mode of transmission and some other factors.
+
+A typical networking setup requires two types of devices: **Servers and Workstations.**
+Servers are the computers responsible for powering the network. Workstations are the devices that make up the network. Some of those devices could be printers, laptops, smartphones, etc. Data transfer and resource sharing is done from the servers. The shared materials are downloaded on the workstations in the form of data packets. While some can be used across countries, others can barely be used within a building. Here are some of the most popular types of networks in use.
+
+### Network Types
+#### Local Area Network (LAN)
+Computer networks that are restricted to a small localized area are called [LANs](https://en.wikipedia.org/wiki/Local_area_network). These types of networks are used to share resources and transfer data within buildings such as schools, offices, net cafes, etc. Routers, Switches, hubs are some of the devices used to operate LAN. All the devices that are powered by LAN, are near to each other and present in the same room or building.
+Due to this reason, cables are used as a transmission medium. Ethernet cables are supported by many devices that have the same interface. They are also the most commonly used cables for connecting modems, routers, etc. Their signal strength is consistent and the communication network is also stable. Thus making them a practical choice for using wired LAN.
+
+![Example of LAN](/engineering-education/network-types-topologies/lan.jpg)
+
+#### Wireless Local Area Network (WLAN)
+[WLAN(Wireless LAN)](https://en.wikipedia.org/wiki/Wireless_LAN) works the same way as the LAN. The network spread is across a limited area. But this type of network does not require any cables or optic fibers. Unlike the LAN, here the mode of transmission is wireless. For wireless transmission, [Wi-Fi(Wireless Fidelity)](https://en.wikipedia.org/wiki/Wi-Fi) is used. Workstations can be connected to the internet with the help of Wi-Fi Routers and Adapters. Devices that support wireless transmission can also share data from their memory drives. Some of the devices that support wireless transmission are smartphones, smart-watches, speakers, laptops etc.
+
+![Example of WLAN](/engineering-education/network-types-topologies/wlan.jpg)
+
+#### Metropolitan Area Network (MAN)
+The point of having [MANs](https://en.wikipedia.org/wiki/Metropolitan_area_network) is similar to LANs. One of the differences is the geographical spread of the network range. As mentioned above, LANs are confined to small areas. But, as the name suggests, MAN’s are spread across an entire metropolitan city. They can be used to share data from one point of the city to another. Business establishments, telephone companies, cable TV’s use this type of networks.  

```

Comment:
I think a bit more on MAN would help - as a reader Im left wondering how a MAN is truly different than a LAN - what helps it connect device with such a big range? Is there mini networks in the middle? Is there some hardware involved? 
Any added info to explain how MAN can operate such a big area. 🚀 
is the hardware used more powerful?

Reply:
I've included the transmission methods and the devices used in a MAN. Also provided a link to Modems.